pkg/logging/logger.go

```diff
@@ -235,8 +235,8 @@ func (l *logrusEntryWrapper) Panicf(format string, args ...interface{}) {
 	l.e.Panicf(format, args...)
 }
 
-func (*logrusEntryWrapper) IsTracing() bool {
-	return logrus.IsLevelEnabled(logrus.TraceLevel)
+func (l *logrusEntryWrapper) IsTracing() bool {
+	return l.e.Logger.Level >= logrus.TraceLevel
```

Comment:
This is really weird, I know.  But d29d82e02a21f7f3ce38b016fd84a0822902e1ac changed it to have this global style.  So the existing mostly sorta kinda works, but completely relies on the _global_ `logrus` level.

Maybe do this in a separate PR, so we can have a look *just* at this?